Jenkinsfile

```diff
@@ -284,8 +288,15 @@ pipeline {
     CHANGE_BRANCH = "${env.CHANGE_BRANCH != null ? env.CHANGE_BRANCH : BRANCH_NAME}"
     CHANGE_TARGET = "${env.CHANGE_TARGET != null ? env.CHANGE_TARGET : BRANCH_NAME}"
     CONNECT_PREPROD_URL = 'https://nos-preprod-connect.nuxeocloud.com/nuxeo'
+    // Required by the jx preview command: https://github.com/jenkins-x/jx/blob/8fdc3a1182bc3ed6ae09728721b53ea4fa1d6234/pkg/cmd/preview/preview.go#L1007
+    ORG = "nuxeo"
+    // Required by the jx preview command: https://github.com/jenkins-x/jx/blob/8fdc3a1182bc3ed6ae09728721b53ea4fa1d6234/pkg/cmd/preview/preview.go#L1015
+    APP_NAME = "nuxeo"
+    TEST_NAMESPACE = "$APP_NAME-${BRANCH_NAME.toLowerCase()}"
+    PERSISTENCE = !isPullRequest()
+    DEBUG_OPTION = getDebugOption()
   }
-
+  
```

Comment:
extra spaces

Reply:
found some hand vs code settings to trim extra new lines and trailing spaces.